compass/setup.py

```diff
@@ -1,15 +1,13 @@
 import argparse
 import sys
-import configparser
 import os
 import pickle
 import warnings
 
-from mache import MachineInfo, discover_machine
```

Comment:
No, this is not just clean-up, though I can understand how it might look that way.  Before, we were merging a `ConfigParser` object from `mache` with one from `compass`.  This previous approach has the advantage that it includes some config options set at runtime in `mache` such as the username.  But it doesn't work with the new approach and it isn't important for `compass`.  So I switched to parsing the config files out of `mache` instead of using the `ConfigParser` object, which means that the `MachineInfo` object is no longer needed.